examples/with-next-app/app/providers.tsx

```diff
@@ -53,28 +50,38 @@ const connectors = connectorsForWallets([
   {
     groupName: 'Other',
     wallets: [
-      argentWallet({ projectId, chains }),
-      trustWallet({ projectId, chains }),
-      ledgerWallet({ projectId, chains }),
+      argentWallet({ projectId }),
+      trustWallet({ projectId }),
+      ledgerWallet({ projectId }),
     ],
   },
 ]);
 
 const wagmiConfig = createConfig({
-  autoConnect: true,
```

Comment:
`autoConnect` is deprecated. Users would have to use `useReconnect` hook or specify `reconnectOnMount` prop to `WagmiProvider`. [Link](https://beta.wagmi.sh/react/guides/migrate-from-v1-to-v2#updated-createconfig-parameters)